.github/workflows/scripts/post_before_script.sh

```diff
@@ -2,13 +2,11 @@
 
 set -euv
 
-echo "machine pulp
-login admin
-password password
-" > ~/.netrc
+# Add the QE (public) key to the container env for `test_sign_locally_then_upload_signature`.
+# Please, remove once the verification was made independent of the system keyring.
 
-chmod og-rw ~/.netrc
+cmd_prefix bash -c "curl -L https://github.com/pulp/pulp-fixtures/raw/master/common/GPG-KEY-pulp-qe | gpg --import"
```

Comment:
What's the motivation for this part? Aka, what breaks if this isn't here?

Reply:
The very test mentioned three lines above: `test_sign_locally_then_upload_signature`